src/codegen/luacg.cpp

```diff
@@ -1021,6 +1030,7 @@ void LuaCodeGenerator::GenClassDeclaration(PObjectBase class_obj, bool /*use_enu
 	}
 
 	GetGenEventHandlers( class_obj );
+	m_strUITable = "ui";
```

Comment:
This member should be set only once, it contains the project property of the table name. Here it gets replaced with a hardcoded value and never gets restored